const.js

```diff
@@ -10,6 +13,15 @@ const IPHONE_12_MAX_LONG_SIDE = 926;
 
 const NAVIGATION_HEADER_HEIGHT = 64;
 
+const STATUS_BAR_OFFSET =
+  Platform.OS === 'android' ? -StatusBar.currentConfig : 0;
+const NAVIGATION_BAR_HEIGHT = Device.select({
+  iPhoneX: NAVIGATION_HEADER_HEIGHT + IPHONE_X_NOTCH_PADDING,
+  iPhoneXR: NAVIGATION_HEADER_HEIGHT + IPHONE_XR_NOTCH_PADDING,
+  notchedAndroid: NAVIGATION_HEADER_HEIGHT + StatusBar.currentHeight,
```

Comment:
Not sure if notchedAndroid is resolved properly, I think it applies StatusBar height out of the box?